R/autoplot.R

```diff
@@ -47,8 +47,8 @@ autoplot.epi_df <- function(
     .facet_by = c(".response", "other_keys", "all_keys", "geo_value", "all", "none"),
     .base_color = "#3A448F",
     .max_facets = Inf) {
-  .color_by <- match.arg(.color_by)
-  .facet_by <- match.arg(.facet_by)
+  .color_by <- rlang::match_arg(.color_by)
+  .facet_by <- rlang::match_arg(.facet_by)
```

Comment:
it's `rlang::arg_match` --- `rlang`, `vctrs`, and `purrr` often try to use a convention of putting a sort of generalized "type" descriptor for the main argument as the first "word" in a function name.